content/account/dashboard/api.textile

```diff
@@ -0,0 +1,56 @@
+---
+title: App API keys
+meta_description: “Manage Ably API keys by creating, updating, setting restrictions, and exploring integration options.”
+meta_keywords: "API keys, Ably, create key, update key, key restrictions, integration, webhooks, authentication"
+---
+
+The API keys tab lists all API keys associated with your account and provides information on each key's capabilities and restrictions. You can "create a new API key":#create and manage an API key.
+
+h2(#create). Create a new API key
+
+Click  *create new API key* to create a new API key. During the _create_ process, you can:
+* Assign a friendly name.
+* Give the new API key a descriptive name (e.g. chat app key) so it is easy to identify later.
+
+h2(#manage). Manage an API key
+
+To manage an API key, you set capabilities, define resource restrictions, use revocable tokens for security, and adjust key settings as needed.
+
+h3. Capabilities
+
+Capabilities provide permissions required for managing message flow, user presence, notifications, channel information, and access controls, these depend on what you require the API key to have access to:
+
+|_. Capability |_. Description |
+| *Publish* | Enable clients to publish messages to channels. |
```

Comment:
I know I'm always commenting saying `allow` should be `enable` because it denotes permission, but here it does denote permission, so should be `allow` 😄